src/web/modules/gateway_accounts/gateway_accounts.http.ts

```diff
@@ -166,13 +167,27 @@ const writeAccount = async function writeAccount(req: Request, res: Response): P
     }
   }
 
+  const stripeStatementDetails: {[key: string]: string} = {
+    payoutStatementDescriptor: '',
+    statementDescriptor: ''
+  } 
```

Comment:
Recommend: 
* specify the two values as (maybe?) strings as this describes what to expect, they will only be set given a conditional
* more specific name used throughout

```suggestion
  const stripeAccountStatementDescriptors: {
    payoutStatementDescriptor?: string,
    statementDescriptor?: string
  } = {}
```

Reply:
done